cmd/bridge/main.go

```diff
@@ -122,7 +122,8 @@ func main() {
 
 	fLoadTestFactor := fs.Int("load-test-factor", 0, "DEV ONLY. The factor used to multiply k8s API list responses for load testing purposes.")
 
-	fDevCatalogCategories := fs.String("developer-catalog-categories", "", "Allow catalog categories customization. (JSON as string)")
+	fDevCatalogCategories := fs.String("developer-catalog-categories", "", "DEV ONLY. Allow catalog categories customization. (JSON as string)")
```

Comment:
Is this flag really dev-only? I thought this was used in production (set by the operator).

Reply:
You're right. I removed this again.